lib/sidekiq/processor.rb

```diff
@@ -71,10 +71,6 @@ def inspect
       "<Processor##{object_id.to_s(16)}>"
     end
 
-    def execute_job(worker, cloned_args)
```

Comment:
BTW, I wasn't offended.  I was literally agreeing with you: I wasn't sure I understood the issue either!  :smile: